### ISSUE TITLE
Patch 0.6.0 - access storage bucket deploying on GCP

### ISSUE THREAD

**Issue by aTeoke** [api3dao/airnode#1358]
0.6.0 Patch related to this fix [#1313](https://github.com/api3dao/airnode/issues/1313)

**Comment by Siegrift** [comment]
The PR has been merged, but not yet released.

When @bbenligiray or @aquarat make the release, please ping @kolenic-martin to test this.

**Comment by kolenic-martin** [comment]
Per discussion with @amarthadan and given that @vponline and @Ashar2shahid have tested this extensively (and I have run several successful deployments in all relevant branches for various regions while testing their feature availability), I'm moving this to Done. If anyone wanted some extra particular tests, please ping me; I'll be happy to do them.

**Comment by bbenligiray** [comment]
The actual release will still need to be tested separately

**Comment by kolenic-martin** [comment]
I've successfully tested v0.6.6 and v0.7.3 and both returned sensible output when making requests via the example script and CURL. I tried breaking the config as well in several places (deleted the stage, changed Airnode version, malformed JSON) and all of that was caught by the validator. 

That said, something was going on in the case of v0.6.6 as the first CURL request returned "Could not handle the request". I couldn't reproduce that again, though, as the second request I sent right afterwards (same parameters, everything) was successful. Do we need to investigate further?
